kafka_consumer/datadog_checks/kafka_consumer/legacy_0_10_2.py

```diff
@@ -58,6 +58,12 @@ def __getattr__(self, item):
         except AttributeError:
             raise AttributeError("LegacyKafkaCheck_0_10_2 has no attribute called {}".format(item))
 
+    @property
+    def kafka_client(self):
+        if self._kafka_client is None:
+            self._kafka_client = self._create_kafka_client()
+        return self._kafka_client
```

Comment:
Is this suppose to be `self.kafka_client`?

Reply:
No, that would call itself 😕 . The `@property` is  `self.kafka_client` and the attribute is `self._kafka_client` which is lazily initialised here.